internal/controller/instaslice_controller.go

```diff
@@ -104,12 +103,12 @@ func (r *InstasliceReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	}
 
 	// pod is completed move allocation to deleting state and return
-	if pod.Status.Phase == v1.PodSucceeded && controllerutil.ContainsFinalizer(pod, "org.instaslice/accelarator") {
+	if pod.Status.Phase == v1.PodSucceeded && controllerutil.ContainsFinalizer(pod, acceleratorName) {
```

Comment:
This is both the Finalizer name and gate name, please check PR #91

Reply:
done.